<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Buttons</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>518</width>
             <height>260</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Parent</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QGroupBox" name="groupBox_2">
                 <property name="title">
                  <string>Child 1</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QPushButton" name="pushButton">
                    <property name="text">
                     <string>Click me!</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_3">
                 <property name="title">
                  <string>Child 2</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QPushButton" name="pb_incrementLabel">
                    <property name="text">
                     <string>No! Click me instead!</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget" native="true">
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <item>
                      <widget class="QLabel" name="lb_increment">
                       <property name="text">
                        <string>You clicked %1 time(s)!</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Progress Bars</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>501</width>
             <height>390</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>0%</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>50%</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar_2">
              <property name="value">
               <number>50</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>100%</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar_3">
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Indeterminate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar_4">
              <property name="maximum">
               <number>0</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Custom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QRadioButton" name="rb_pb_indeterminate">
                 <property name="text">
                  <string>Indeterminate</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rb_pb_specificValue">
                 <property name="text">
                  <string>Specific value:</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="sl_pbValue">
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="pb_custom">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Form</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QScrollArea" name="scrollArea_3">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>-258</y>
             <width>501</width>
             <height>518</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item alignment="Qt::AlignHCenter">
             <widget class="QWidget" name="widget_3" native="true">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>First name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="le_firstname">
                 <property name="inputMask">
                  <string/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>Your first name here...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Last Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="le_lastname">
                 <property name="placeholderText">
                  <string>Your last name here...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Age</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="sp_age">
                 <property name="suffix">
                  <string> year(s) old</string>
                 </property>
                 <property name="maximum">
                  <number>200</number>
                 </property>
                 <property name="value">
                  <number>18</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Birthday</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateTimeEdit" name="dt_birthday"/>
               </item>
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Gender</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cb_gender">
                 <property name="editable">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lb_text_pronoun">
                 <property name="text">
                  <string>Pronoun</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cb_pronouns">
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="gb_pronounExample">
                 <property name="title">
                  <string>Example</string>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_12">
                  <item>
                   <widget class="QLabel" name="lb_pronounExample">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>100</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Please fill `first name` and `pronoun` fields first</string>
                    </property>
                    <property name="scaledContents">
                     <bool>true</bool>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
